rename_files.py

```diff
@@ -33,29 +33,35 @@ def main():
 
 
 def get_input():
+    print(" You entered", len(sys.argv)-1, "arguments at the command line.")
+
+    if len(sys.argv) != 4:
+        # IndexError: list index out of range
+        print(" Error: Wrong number of arguments. Enter 3 arguments: 1. "
+              "string to replace 2. replacement string 3. path for files ")
+        sys.exit(1)
+
```

Comment:
Oh, I see that your changes include the ones for your other PR #3. If you merge that one first these changes will disappear.

I'll show you tomorrow how to avoid including those changes.